rubin_jupyter_utils/hub/scanrepo/scanrepo.py

```diff
@@ -35,6 +37,8 @@ def __init__(
         insecure=False,
         sort_field="name",
         debug=False,
+        username=None,
```

Comment:
Again, repo_username etc. would make intent a little more clear, but maybe there's not much risk of confusion if you're already inside the scanner.